narwhals/dataframe.py

```diff
@@ -581,8 +584,15 @@ def to_pandas(self) -> pd.DataFrame:
     def write_csv(self, file: str | Path | BytesIO | None = None) -> Any:
```

Comment:
The return type here should be `str | None` from my understanding. I changed this but then got mypy errors. I started digging into the issue but then decided to leave it like it is for now in favour of getting the documentation done. I can open an issue for it if you like.

Reply:
Sure thanks!